docs/dev/build.md

```diff
@@ -291,35 +342,39 @@ This can be done using two ways:
 - Compile on Intel CPU host using native compilation. Note, that [Build steps](#build-steps) show this scenario.
 - Cross-compile on OSX Apple Silicon.
 
+The software was validated on:
+- macOS 10.x, 11.x, 12.x x86 64-bit
+- macOS 11.x, 12.x, arm64 (cross-compilation)
+
 1. Create a build folder:
 ```sh
-mkdir build && cd build
+%  mkdir build && cd build
```

Comment:
In previous snippets we don't have `%` I propose to have one style for snippets

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/562c6c32912eb936d9ca384c7caa9cf772ce2fa2